lxd/main_import.go

```diff
@@ -15,8 +15,8 @@ import (
 func cmdImport(args []string) error {
 	name := args[1]
 	b := shared.Jmap{
-		"target": name,
-		"force":  *argForce,
+		"name":  name,
+		"force": *argForce,
```

Comment:
Can this whole commit be merged with the original --force commit to make this set a bit cleaner?